python/lsst/pipe/supertask/config.py

```diff
@@ -0,0 +1,94 @@
+#
+# LSST Data Management System
+# Copyright 2018 LSST Corporation.
```

Comment:
It's definitely not LSST Corp. Also, it might actually be SLAC in this case (in theory you are meant to put your home institution on copyright). Finally, LSST is not a copyright owning entity and it should really be the expanded AURA not "AURA" that is in the copyright statement. Other than that, it's close and "AURA/LSST" is at least consistent with what we incorrectly do in other code...

Reply:
I guess that is from copy&paste, indeed bunch of files contain the same "LSST Corporation", I'm going to change them all in a separate commit.